.github/workflows/linux.yml

```diff
@@ -138,8 +138,16 @@ jobs:
           restore-keys: package-${{ matrix.os }}-${{ matrix.package }}-ccache-
       - name: Build with Docker
         run: |
+          case ${{ matrix.os }} in
+            debian-*)
+              deb_release=$(head -n1 packages/${{ matrix.package }}-mroonga/debian/changelog | \
+                            awk '{print $2}' | \
+                            awk -F "-" '{print $2}' | \
+                            sed -e 's/[^0-9]//g')
+              ;;
+          esac
           cd packages/${{ matrix.package }}-mroonga
-          rake ${{ matrix.package-type }}:build BUILD_DIR=build
+          rake ${{ matrix.package-type }}:build BUILD_DIR=build DEB_RELEASE=${deb_release}
```

Comment:
Why do we need to specify `DEB_RELEASE` here?
`DEB_RELEASE` is for `rake version:update` not `rake apt:build`.